lib/pf/util.pm

```diff
@@ -751,16 +751,18 @@ sub pf_run {
     my $caller = ( caller(1) )[3] || basename($0);
     $caller =~ s/^(pf::\w+|main):://;
 
+    my $loggable_command = $command;
+    $loggable_command =~ s/$options{log_strip}/*obfuscated-information*/g; 
```

Comment:
That was me talking anyway. 
But for example if I'd like to do a common regex to strip any MySQL password instead of knowing the password to strip I could. That's mainly why I left it as an interpretable regex